integration_tests/serverless.yml

```diff
@@ -199,6 +199,22 @@ functions:
       DD_EXTRA_TAGS: tagC:valueC tagD:valueD
       DD_LAMBDA_HANDLER: src/logPython.log
 
+  log-go:
+    without-ddlambda-go:
```

Comment:
This looks a bit off, what's going on here?

Reply:
i.e. without-dd-lambda go nested under log-go, maybe I am missing something though